src/errors/WalletInvalidData.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+
+class WalletInvalidDataError extends Error {
```

Comment:
better to call it `InconsistentData`
and message is not needed for this type of error